### PR TITLE
 Support multi-line questions

### LINKED ISSUE #10: Support multi-line questions
Seems like the `create-card` command doesn't consider `\n` in the question's input, but I'd very much like to have them.

i.e. `create-card -q "foo\nbar"` should create a question with two lines. The review section should be updated as well with regards to this change.

### PR DESCRIPTION
**Pull request by resolritter**
close #10

### CODE REVIEW

File: drillsrs/cmd/study.py [hunk 1]
```diff
@@ -21,10 +22,7 @@ def _learn_single_card(
     if mode is Mode.reversed or mode is Mode.mixed and random.random() > 0.5:
         raw_question, raw_answers = random.choice(raw_answers), [raw_question]
 
-    question = 'Question: %s' % raw_question
-    if card.tags:
-        question += ' [%s]' % util.format_card_tags(card.tags)
-    util.ask(question)
+    print_question(raw_question, raw_answers, card.tags)
```
Comment (rr-) [inline drillsrs/cmd/study.py:25]: This change causes the script to show answers immediately during the study session, which is not desired.

Comment (resolritter) [reply drillsrs/cmd/study.py]: Sorry! Tried to fix it :-v

File: drillsrs/question.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+def render_question(raw_question, raw_answers, tags):
+    question_header_template = "Question: %s"
+
+    if util.is_raw_multiline(raw_question):
+        print(
+            question_header_template
+            % ("[%s]" % util.format_card_tags(tags) if tags else "")
+        )
+        return util.from_raw_to_multiline(raw_question)
```
Comment (rr-) [inline drillsrs/question.py:12]: Now this both prints and returns a string

Comment (resolritter) [reply drillsrs/question.py]: I'm back! I've tried to solve it by putting everything in a single string

> Now this both prints and returns a string, it should do either one thing or another

This wasn't noticeable in the test runs I did... My bad

**Review by rr- — APPROVED** [review]

**Comment by rr-** [comment]
Thanks, merged

**Pull request merged**